Sources/SpeziQuestionnaire/ORKOrderedTaskView.swift

```diff
@@ -35,10 +35,10 @@ struct ORKOrderedTaskView: UIViewControllerRepresentable {
                 
                 switch reason {
                 case .completed:
-                    let fhirResponse = taskViewController.result.fhirResponse
-                    fhirResponse.subject = Reference(reference: FHIRPrimitive(FHIRString("My Patient")))
+                    let response = taskViewController.result
                     
-                    await questionnaireResponse(fhirResponse)
+                    // what is this doing down here?
+                    await timedWalkResponse(response)
```

Comment:
I am unsure if I'm doing this correctly here. Is it now up to the developer using our Spezi framework to do what they want with the response?

Reply:
Also, this is the incorrect type with error message "cannot convert value of type 'ORKTaskResult' to expected argument type 'ORKTimedWalkResult'".

I also tried casting it in line 38 above, but they are incompatible. I might need a little refresher on class inheritance (both ORKTaskResult and ORKTimedWalkResult are subclasses of ORKResult, but I am unsure how to syntactically convert them)